elasticsearch/sgconfig/roles_mapping.yml

```diff
@@ -38,5 +38,5 @@ admin_user:
 
 project_user:
   readonly: true    
-  users:    
-    - '*'
\ No newline at end of file
+  backendroles:
```

Comment:
not sure we need to do this based on https://github.com/openshift/elasticsearch-proxy/pull/39#issuecomment-634960510